pkg/controllers/management/drivers/nodedriver/machine_driver.go

```diff
@@ -86,7 +87,9 @@ func (m *Lifecycle) download(obj *v3.NodeDriver) (*v3.NodeDriver, error) {
 		existingSchema, err = m.schemaLister.Get("", schemaName)
 	}
 
-	if driver.Exists() && err == nil {
+	anno := obj.Annotations[driverSchemaRefresh]
+
+	if driver.Exists() && err == nil && anno == "true" {
```

Comment:
not really an acceptable solution as it wont allow for future updates of drivers. We need to permanently fix node drivers so that they update on upgrade.
Here's what I think we can/should do: 
Create a new field on nodeDriverStatus called applied version. It will capture what the last version of the driver we downloaded and installed was. For built-in drivers, you can use `docker-machine --version`. For third-party drivers, we can use the calculated hash after download (not the user supplied hash bc that is optional)

Next time this runs, recalculate the appliedVersion (by calling docker-machine --version or hashing the downloaded driver again). If it differs from the appliedHash on the status object, then update the dynamic schema.